connection/connection_params.ts

```diff
@@ -47,19 +47,26 @@ export interface TLSOptions {
   /**
    * If TLS support is enabled or not. If the server requires TLS,
    * the connection will fail.
+   *
+   * Default: `true`
    */
   enabled: boolean;
   /**
    * This will force the connection to run over TLS
    * If the server doesn't support TLS, the connection will fail
    *
-   * default: `false`
+   * Default: `false`
    */
   enforce: boolean;
   /**
-   * A custom CA file to use for the TLS connection to the server.
+   * A list of root certificates that will be used in addition to the default
+   * root certificates to verify the server's certificate.
+   *
+   * Must be in PEM format.
+   *
+   * Default: `[]`
    */
-  caFile?: string;
+  caCerts?: string[];
```

Comment:
No need to make it optional, since the signature used for configuration is Partial<TLSOptions> and you are already defaulting to an empty array